3-Python

dahachm opened the pull request:
@Cakemanator

**3-Python**

9 tasks to practise python skills.

All the comments and specifications about the task in `task_*` directories.

dahachm committed:
Add task_1

dahachm committed:
Add task_2

dahachm committed:
Add task_3

dahachm committed:
Add task_4

dahachm committed:
Add task_5

dahachm committed:
Add task_6

dahachm committed:
Add task_7

dahachm committed:
Add task_8

dahachm committed:
Create README.md for task_1

dahachm committed:
Create README.md for task_2

dahachm committed:
Create README.md for task_3

dahachm committed:
Create README.md for task_4

dahachm committed:
Create README.md for task_5

dahachm committed:
Create README.md task_6

dahachm committed:
Create README.md for task_7

dahachm committed:
Create README.md for task_8 (AWK 1)

dahachm committed:
Create README.md for task_8 (AWK 2)

dahachm committed:
Create README.md for task_8 (AWK 3)

dahachm committed:
Create README.md for task_8 (AWK 4)

dahachm committed:
Add task_9

dahachm committed:
Update __init__ in Docker

	In making remoteHost str suitable for `requests_unixsocket` class:

	- fix updating remoteHost value after colling .replace('/', '%2F')

	- add '//' after 'unix:' in remoteHost str after calling
	.replace('/', '%2F')

dahachm committed:
Create README.md for task_9

Cakemanator commented on 3-Python/task_2/task_2.py:
```diff
@@ -0,0 +1,49 @@
+
+print('\n    --- Case #5 ---')
+print(f'Input: {test_5} (type: {type(test_5)})')
+printEven(test_5)
```
What if I want to provide my own set of numbers?

dahachm replied:
Hi, @Cakemanator!

Thank you for feedback!

I added an option to input user's list

Cakemanator commented on 3-Python/task_3/task_3.py:
```diff
@@ -0,0 +1,37 @@
+
+    secret = input('Tell me your secret (word/phrase): ')
+
+    if secret in dictionary:
+        print('Nice! Your secret matches.')
+        exit()
+    elif secret in dictionary:
+        print('Nice! Your secret matches.')
+        exit()
```
Why do you repeat exact same condition?

dahachm replied:
Looks like a result of copy-paste process, can't remember what was the point, tbh :D 

I removed this repeated condition

Cakemanator commented on 3-Python/task_5/task_5.py:
```diff
@@ -0,0 +1,63 @@
+print('Input: ' + str(test_5))
+top, extras = top3Max(test_5)
+print (top)
+print (extras)
```
What if I want to provide my own set of numbers?

dahachm replied:
Also added an option for user to input his list!

dahachm committed:
Update task_2.py

        Add ability to input user`s list of numbers

        In printEven:
                now continue to process input list even if non-number
                value is met
                count non-number values and print its total number

dahachm committed:
Udate task_3

	Remove repeated condition when checking the value of the secret

dahachm committed:
Update task_5

	Add ability to input user`s list of numbers

dahachm committed:
Remove debug output :)
